app/policies/network_membership_policy.rb

```diff
@@ -5,14 +5,12 @@ def create?
 
   # create or update by network owner account admin
   def update?
-    token && token.authorized?(resource.network.account.id, :admin)
+    token&.authorized?(resource.network.account.id, :network_members)
   end
 
   # destroy by network owner account admin, or by network member account admin
   def destroy?
-    token && (
-      token.authorized?(resource.network.account.id, :admin) ||
-      token.authorized?(resource.account.id, :admin)
-    )
+    token&.authorized?(resource.network.account.id, :network_members) ||
+      token&.authorized?(resource.account.id, :network_members)
```

Comment:
This feels weird to me - I think network-members is being a little overloaded but I don't have a better suggestion in the current list of scopes - @cavis thoughts?